src/ports/postgres/dbconnector/TypeTraits_impl.hpp

```diff
@@ -750,6 +784,30 @@ struct TypeTraits<dbal::eigen_integration::SparseColumnVector> {
     );
 };
 
+template <>
+struct TypeTraits<std::vector<uint16_t> > {
+    typedef std::vector<uint16_t> value_type;
+
+    WITH_OID( INT4ARRAYOID );
+    WITH_MUTABILITY( dbal::Mutable );
+    WITH_DEFAULT_EXTENDED_TRAITS;
+    // FIXME a quick hack
```

Comment:
Could we add a bit more details in the comment saying the cons of this hack and a potential direction for a future fix?